examples/regression/4-tutorials/plot_ccp_tutorial.py

```diff
@@ -0,0 +1,509 @@
+mapie_ccp_3.fit(X_train, y_train)
+y_pred_ccp_3, y_pi_ccp_3 = mapie_ccp_3.predict(X_test)
+
+mapies = [mapie_split, mapie_cv, mapie_cqr,
+          mapie_ccp_1, mapie_ccp_2, mapie_ccp_3]
+y_preds = [y_pred_split, y_pred_cv, y_pred_cqr,
+           y_pred_ccp_1, y_pred_ccp_2, y_pred_ccp_3]
+y_pis = [y_pi_split, y_pi_cv, y_pi_cqr, y_pi_ccp_1, y_pi_ccp_2, y_pi_ccp_3]
+titles = ["Basic Split", "CV+", "CQR", "CCP 2 groups, 1 and 1",
+          "CCP 2 groups, 1 and X", "CCP 2 groups, polynomials"]
+
+plot_figure(mapies, y_preds, y_pis, titles)
+plot_widths(titles, y_pis)
+
+##############################################################################
+# 5.4. Improve the performances without prior knowledge:
+# :class:`~mapie.calibrators.GaussianCCP`
+# --------------------------------------------------------------------------
+# We can use :class:`~mapie.calibrators.GaussianCCP` calibrators,
+# if we don't have prior information
+# about the data. It will sample points (are use the points given by the user),
+# and only consider the calibration conformity scores of points next to a
+# sample, to estimate the prediction interval of this sample. In this way,
+# assuming wehave enough points, and the correct standard deviation value,
+# we will getan overall good adaptativity.
+
+calibrator_gauss1 = GaussianCCP(np.arange(-1, 6).reshape(-1, 1), 1)
+calibrator_gauss2 = GaussianCCP(30, 0.05, random_sigma=True, normalized=True)
+calibrator_gauss3 = GaussianCCP(30, 0.25, random_sigma=True, normalized=True,
+                                reg_param=1e-3)
+
+# ================== CCP 1  ==================
+mapie_ccp_1 = SplitCPRegressor(estimator, calibrator=calibrator_gauss1,
+                               alpha=ALPHA, random_state=random_state)
+mapie_ccp_1.fit(X_train, y_train)
+y_pred_ccp_1, y_pi_ccp_1 = mapie_ccp_1.predict(X_test)
+
+# ================== CCP 2  ==================
+mapie_ccp_2 = SplitCPRegressor(estimator, calibrator=calibrator_gauss2,
+                               alpha=ALPHA, random_state=random_state)
+mapie_ccp_2.fit(X_train, y_train)
+y_pred_ccp_2, y_pi_ccp_2 = mapie_ccp_2.predict(X_test)
+
+# ================== CCP 3  ==================
+mapie_ccp_3 = SplitCPRegressor(estimator, calibrator=calibrator_gauss3,
+                               alpha=ALPHA, random_state=random_state)
+mapie_ccp_3.fit(X_train, y_train)
+y_pred_ccp_3, y_pi_ccp_3 = mapie_ccp_3.predict(X_test)
+
+mapies = [mapie_split, mapie_cv, mapie_cqr, mapie_ccp_1,
+          mapie_ccp_2, mapie_ccp_3]
+y_preds = [y_pred_split, y_pred_cv, y_pred_cqr, y_pred_ccp_1,
+           y_pred_ccp_2, y_pred_ccp_3]
+y_pis = [y_pi_split, y_pi_cv, y_pi_cqr, y_pi_ccp_1, y_pi_ccp_2, y_pi_ccp_3]
+titles = ["Basic Split", "CV+", "CQR", "CCP, 5 points, s=1 (under-fit)",
+          "CCP, 30 points, s=0.05 (over-fit)",
+          "CCP, 30 points, s=0.25 (good calibrator)"]
+
+plot_figure(mapies, y_preds, y_pis, titles, show_transform=True)
+plot_widths(titles, y_pis)
```

Comment:
It's a shame we don't see the conditional coverage here? Why did we choose not to show this?